packages/integrations/image/components/index.ts

```diff
@@ -2,6 +2,54 @@
 export { default as Image } from './Image.astro';
 export { default as Picture } from './Picture.astro';
 
+import type { TransformOptions, OutputFormat } from '../dist/loaders/index.js';
+import type { ImageMetadata } from '../dist/vite-plugin-astro-image.js';
+
+export interface ImageComponentLocalImageProps
+	extends Omit<TransformOptions, 'src'>,
+		Omit<ImgHTMLAttributes, 'src' | 'width' | 'height'> {
```

Comment:
Should be `astroHTML.JSX.ImgHTMLAttributes` too?

Reply:
Maybe `HTMLAttributes<'img'>` from [`astro/types`](https://github.com/withastro/astro/blob/main/packages/astro/types.d.ts#L7)?

Reply:
Sounds good to me!
So we could refactor [ImgHTMLAttributes and  HTMLAttributes](https://github.com/withastro/astro/pull/5788/files#diff-10f5ca07e9c63bb301d686880dfcb4497bd60f41a1f6d64a168f9950f960c7d4R54-R61) too perhaps?

Reply:
> Maybe `HTMLAttributes<'img'>` from [`astro/types`](https://github.com/withastro/astro/blob/main/packages/astro/types.d.ts#L7)?

Sounds good to me!
So we could refactor [ImgHTMLAttributes and  HTMLAttributes](https://github.com/withastro/astro/pull/5788/files#diff-10f5ca07e9c63bb301d686880dfcb4497bd60f41a1f6d64a168f9950f960c7d4R54-R61) too perhaps?

Reply:
> Should be `astroHTML.JSX.ImgHTMLAttributes` too?

Probaly not? I don't think we want directives here